Views/UserManagement/Users.cshtml

```diff
@@ -18,7 +17,14 @@
             <tr>
                 <td>@Html.DisplayFor(model => user.UserName)</td>
                 <td>@Html.DisplayFor(model => user.Email)</td>
-                <td>@String.Join(",", await userManager.GetRolesAsync(user))</td>
+                @{
+                    var roleList = new List<string>();
```

Comment:
create this variable (roleList) in controller and pass to view through a ViewModel